### PR TITLE
Various updates to Los Angeles January 2016

### PR DESCRIPTION
**Pull request by irabinovitch**
Updating Los Angeles 2016 formatting and content to match our existing site and formatting, now that we're being listed on DevOpsDays.org as well.

TODO:
* add registration info
* add remaining sponsors

### CODE REVIEW

File: site/content/events/2016-losangeles-1day/_email_organizers.txt [hunk 1]
```diff
@@ -1 +1 @@
-organizers-losangeles-2016@devopsdays.org
+devopsdayla-planning@googlegroups.com
```
Comment (phrawzty) [inline site/content/events/2016-losangeles-1day/_email_organizers.txt]: My understanding is that the devopsdays.org MLs are preferred; maybe @KrisBuytaert or @bridgetkromhout can confirm the policy (if any) here.

Comment (irabinovitch) [inline site/content/events/2016-losangeles-1day/_email_organizers.txt]: I understand, but we've been running the event with the google group for 6 years now and have our organizers on it as well as our archive history.  It's also been published as part of our call for papers on DevOps.com and a number of other media sites.

If/when  we move to a 2 day event for April, we'll move back to a devopsdays.org address.

Comment (phrawzty) [inline site/content/events/2016-losangeles-1day/_email_organizers.txt]: :+1:

**Comment by irabinovitch** [comment]
Thanks!

**Pull request merged**